### ISSUE TITLE
archive.org

### ISSUE THREAD

**Issue by saadbruno** [anudeepND/whitelist#41]
archive.org and web.archive.org are both blocked by the tspprs list. So it would be nice to have on the whitelist script

**Comment by anudeepND** [comment]
@saadbruno I have contacted the maintainer of that host file over reddit, he will be removing it in the next release

**Comment by anudeepND** [comment]
I will be closing this since it's been removed from the list and these domains are not present in other lists
